timer_node:
  ros__parameters:
    IP: "192.168.2.50"
    local_interface_ip: "192.168.2.1"
    speed_of_sound: 1445
